dts/bindings/sensor/st,stm32-temp-cal.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2022, Wouter Cappelle
+# SPDX-License-Identifier: Apache-2.0
+
+description: STM32 family TEMP node for production calibrated sensors like L5/U5
+
+compatible: "st,stm32-temp-cal"
+
+include: [base.yaml, "st,stm32-temp-common.yaml"]
+
+properties:
+    ts-cal1-addr:
+      type: int
+      default: 0x0BFA05A8
```

Comment:
Please remove default (no reason to privilege L5 over other series), apply everywhere.

Reply:
done, move them to the l5 board overlay & made the fields required